cmd/main.go

```diff
@@ -51,6 +52,11 @@ var (
 	setupLog = ctrl.Log.WithName("setup")
 )
 
+const (
+	unableToCreateControllerString = "unable to create controller"
```

Comment:
maybe drop the `String` prefix from var names

Reply:
Not required anymore as this is currently inline with the error.